api/pkg/service/resource/resource.go

```diff
@@ -20,15 +20,15 @@ import (
 	"strings"
 
 	"github.com/jinzhu/gorm"
-	"go.uber.org/zap"
+	"github.com/tektoncd/hub/api/gen/log"
 
 	"github.com/tektoncd/hub/api/gen/resource"
 	"github.com/tektoncd/hub/api/pkg/app"
 	"github.com/tektoncd/hub/api/pkg/db/model"
 )
 
 type service struct {
-	logger *zap.SugaredLogger
+	logger *log.Logger
 	db     *gorm.DB
```

Comment:
we can delete this db as well - right? 
None of the services would now need to have `db` - right?

Reply:
I was yet to update the pr.. you reviewed early I guess :smile: